packages/formik/test/withFormik.test.tsx

```diff
@@ -103,6 +103,7 @@ describe('withFormik()', () => {
       setSubmitting: expect.any(Function),
       setTouched: expect.any(Function),
       setValues: expect.any(Function),
+      isFormValid: expect.any(Function),
```

Comment:
isValid?

Reply:
isFormValid is a function passed by Formik which is needed to calculate whether the form is valid. Otherwise, we need to pass a bunch of config props from Formik through the API, and if a user doesn't memoize them we lose all of the performance gains.

Reply:
Removed isFormValid in favor of exposing formikApi.useComputedState().